src/platform/testing/unit/mocha-setup.js

```diff
@@ -63,7 +63,15 @@ export default function setupJSDom() {
 
   win.dataLayer = [];
   win.scrollTo = () => {};
-  win.sessionStorage = {};
+  Object.defineProperty(win, 'localStorage', {
```

Comment:
Why do we need to use `defineProperty` here?

Reply:
Same reason we need to do it for `sessionStorage`. Changes to JSDOM prevent us from manipulating that property unless we reset it to something we can overwrite which is something we do in some of our tests.